research/lamp-automated-model-parallelism/data_utils.py

```diff
@@ -53,7 +53,7 @@ def load_data_and_mask(data, mask_data):
     img = np.load(data)  # z y x
     img = pad_xform(img[None])[0]
     item = dict(image=img, label=[])
-    for idx, maskfnm in enumerate(mask_data):
+    for _, maskfnm in enumerate(mask_data):
```

Comment:
> B007 Loop control variable '{}' not used within the loop body. If this is intended, start the name with an underscore.

Should I rewrite the loop if it has redundancies or is it better to leave it

```python
for maskfnm in mask_data:
```

Reply:
yes please change this, thanks!